Jenkinsfile

```diff
@@ -76,3 +95,15 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
+
+void renameFiles(def files, String prefix) {
+    for(i = 0; i < files.length; i++) {
```

Comment:
As the script is evaluated on the Jenkins master, `File` cannot be used at all -- it knows nothing about remoting. `FilePath`, maybe, but that can probably be abused as well.